nix-tools-default.nix

```diff
@@ -30,23 +39,23 @@ in {
     nix-tools = { _raw = nix-tools; }
```

Comment:
Do we still need this, if we have nix-tools-raw?

Reply:
I left it in for backwards compatibility.

Reply:
note that backward compatibility is already "broken" by removing `nix-tools-pkgs-path`.

Reply:
on that note, please update `changelog.md` so that people can easily make the necessary changes after updating iohk-nix in their project.